duckduckgo/iOS
# Separate test userdefaults from main app

Task/Issue URL: https://app.asana.com/0/414709148257752/1201758755405316/f

**Description**:
Change how we access UserDefault.standard so when running tests it doesn't affect the main app

**Steps to test this PR**:
1. Run app normally
2. Run the tests
3. Run the app again and check if the app crashes due to wrong migrations

-------
1. Run app normally and change the theme
2. Run the tests
3. Run the app again and check if the theme is still the same4. 

------
1. Test storage/settings, if they are persisted as it should.
2. Run smoke test3. 

**OS Testing**:
